docs/docs/60-new-docs/50-user-guide/20-how-to-guides/16-verification.md

```diff
@@ -0,0 +1,215 @@
+:::
+
+
+##  Arguments and Metadata
+
+It is also possible to specify additional labels, annotations, and arguments
+that should be applied to `AnalysisRun` resources spawned from the referenced
+`AnalysisTemplate`:
+
+```yaml
+apiVersion: kargo.akuity.io/v1alpha1
+kind: Stage
+metadata:
+  name: test
+  namespace: kargo-demo
+spec:
+  # ...
+  verification:
+    analysisTemplates:
+    - name: kargo-demo
+    analysisRunMetadata:
+      labels:
+        foo: bar
+      annotations:
+        bat: baz
+    args:
+    - name: foo
+      value: bar
+```
+
+## Implicit Argo CD Verification
+
+It is usually the desire to wait until a deployment is fully rolled out and updated, before performing any testing and verification. For example, it would be premature to start running tests if a Deployment's Pods were still coming up and or not fully updated to the desired version. Promotions might only include the apply/sync of new manifests to the cluster, but do not not necessarily wait until the rollout was fully complete (which might take minutes depending on many factors like Pod startup time).
+
+In order to prevent verification from starting before Applications are ready, Kargo has a built in understanding of Argo CD health, and will delay beginning verification until the Application had reached a Healthy state.
+
+If a Stage references one ore more Argo CD Applications as part of its promotion steps, after Promotion, Kargo will automatically wait until the Argo CD Applications reach a `Healthy` state. This is an built-in, "implicit" form of verification that is not configured explicitly in the `spec.verifications`. After the Applications reaches the `Healthy` state, Kargo will then run the specified verifications in `spec.verifications` (if any). This ensures that testing begins only after all resources of the Application have been successfully updated after the promotion.
+
+
+## Soak Times
+
+In some scenarios, automated testing may be unavailable or insufficient to ensure confidence before promoting software to critical environments. To mitigate this, users often resort to letting the release run in a lower environment for a period of time, monitoring its behavior and stability before proceeding with promotion. This practice is commonly known as 'soaking' or 'baking' the release before it reaches production.
+
+Kargo features a "soak time" option in Stages when it requests Freight from upstream stages. To configure a soak time, a duration string can `spec.requestedFreight[].requiredSoakTime`. Valid durations are in seconds, minutes or hours (e.g. `180s`, `30m`, `48h`). This duration is the minimum duration (beginning after successful promotion), for which the requested Freight must have continuously occupied ("soaked in") in an upstream Stage before becoming available for promotion to this Stage. Automated and manual promotions would be blocked until the soakTime requirement was met.
```

Comment:
> [..] and manual promotions would be blocked [..]

Should we hint here that it can always be overwritten using manual _approval_? I think people otherwise may think there is no way out if you want to.

Reply:
I did add that below this section.